### ISSUE TITLE
Models trained with Huber and VGG loss

### ISSUE THREAD

**Issue by aravind-anantha** [tarun005/FLAVR#22]
Hi Tarun,

Your work is really interesting! 
I was wondering if you could share the models trained in the ablation experiments. 

I am curious to see how the different loss functions affect downstream tasks like actuon recognition. From the paper, you mentioned that L1 loss results in sharper images. But, does this also translate to better action recognition results?

Please do share any experiments/insights regarding this. I would love to hear your thoughts.

**Comment by tarun005** [comment]
The results shown in the paper are using the best possible model for downstream action recognition. We found the performance is not that different for different choices of loss functions used in the interpolation network.

**Comment by aravind-anantha** [comment]
Thanks for clearing that up.
